bin/server.js

```diff
@@ -56,7 +55,6 @@ const properties = new Properties();
 const sources = new Sources(properties);
 
 // Add metadata and some statics
-properties.dynamic(new Consul('consul', Config.get('consul')), 'consul');
```

Comment:
When do we create the Consul source? I can't find any other references to `new Consul()` where the config would get loaded.